uint16 infraFR
uint16 infraFL
uint16 infraBR
uint16 infraBL
uint16 infraSR
uint16 infraSL
